plugins/ProfileExtender/views/registrationfields.php

```diff
@@ -12,6 +12,10 @@
             $Options = array_combine($values, $labels);
         }
 
+        if ($Field['FormType'] == 'TextBox' && !empty($Field['Options'])) {
```

Comment:
Was there a specific reason we decided to do this only for text boxes and not other fields?

Reply:
Just added a comment on the main issue:

> This fix replicate exactly what is in the other view to minimize unforeseen problems. The logic seems very odd to me...

Reply:
That's fair.